src/placeos-rest-api/controllers/root.cr

```diff
@@ -37,19 +39,13 @@ module PlaceOS::Api
     end
 
     post "/reindex", :reindex do
-      response = HTTP::Client.post(
-        "http://rubber-soul:3000/api/rubber-soul/v1/reindex?backfill=#{params["backfill"]? == "true"}",
-        headers: HTTP::Headers{"X-Request-ID" => request_id},
-      )
-      head response.status_code
+      RubberSoul::Client.client &.reindex(backfill: params["backfill"]? == "true")
```

Comment:
Does the client raise an error here if the request fails?

Reply:
Yep